src/slam_toolbox_common.cpp

```diff
@@ -844,6 +886,8 @@ LocalizedRangeScan * SlamToolbox::addScan(
     dataset_->Add(range_scan);
 
     publishPose(range_scan->GetCorrectedPose(), covariance, scan->header.stamp);
+    publishNewNodeEvent(range_scan);
+    requestPoseGraphPublish();
```

Comment:
Do you think this should happen on each new node or only when a loop closure happens? This could become quite a large thing to publish every ~second as this grows to a very large space

Reply:
@SteveMacenski We were publishing the full pose_graph on every new node event because I thought other users might expect the topic to be updated more frequently in general cases.
However, I agree with your point. The full graph takes up a lot of space, and incremental updates using new node events alone should be sufficient. Also, handling general cases is beyond the scope of this PR.

We'll remove the full pose_graph publication from every new node event, and update the README to clarify that the pose_graph topic is only published on loop closures.